writePass/index.js

```diff
@@ -136,10 +154,25 @@ exports.handler = async (event, context) => {
       const res = await dynamodb.putItem(passObject).promise();
       console.log("res:", res);
 
-      // TODO: ***SEND EMAIL TO CONFIRM
-
-
-      return sendResponse(200, AWS.DynamoDB.Converter.unmarshall(passObject.Item));
+      const emailRes = await axios({
+        method: 'post',
+        url: process.env.GC_NOTIFY_API_PATH,
+        headers: {
+          'Authorization': process.env.GC_NOTIFY_API_KEY,
+          'Content-Type': 'application/json'
+        },
+        data: {
+          'email_address': email,
+          'template_id': gcNotifyTemplate,
+          'personalisation': personalisation
+        }
+      });
+
+      if (emailRes.status === 201) {
+        return sendResponse(200, AWS.DynamoDB.Converter.unmarshall(passObject.Item));
+      } else {
+        return sendResponse(400, { msg: 'Email Failed to Send' });
+      }
```

Comment:
I don't think we want to fail here because the pass was indeed created.  Just something wrong with the email service.